src/Felix/Equiv.agda

```diff
@@ -32,13 +32,17 @@ record Equivalent q {obj : Set o} (_⇨_ : obj → obj → Set ℓ)
   module ≈-Reasoning {a b} where
     open SetoidR (≈setoid a b) public
 
-  infixr 9 _•_
-  _•_ : {f g h : a ⇨ b} (g≈h : g ≈ h) (f≈g : f ≈ g) → f ≈ h
-  g≈h • f≈g = trans f≈g g≈h
+    infix 2 ⟺
```

Comment:
What is the effect of an `infix` declaration for an operator without one or more underscores in its name?